sdks/csharp/model_operations.cs

```diff
@@ -2,8 +2,7 @@
 // <CreateModel>
 // 'client' is an instance of DigitalTwinsClient
 // Read model file into string (not part of SDK)
-StreamReader r = new StreamReader("MyModelFile.json");
-string dtdl = r.ReadToEnd(); r.Close();
+using var r = new StreamReader("MyModelFile.json");
 string[] dtdls = new string[] { dtdl };
```

Comment:
where is dtdl declared? did you remove line 6 intentionaly?

Reply:
Mistakenly removed the whole line.